### PR TITLE
Document client registration access token rotation executor

### PR DESCRIPTION
**Pull request by reda-alaoui**
For https://github.com/keycloak/keycloak/pull/7955

CC @mposolda

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@reda-alaoui Thanks!

**Comment by mposolda** [comment]
@andymunro Coud you please review this PR?

File: securing_apps/topics/client-registration.adoc [hunk 1]
```diff
@@ -61,7 +61,8 @@ Authorization: bearer eyJhbGciOiJSUz...
 When you create a client through the Client Registration Service the response will include a registration access token.
 The registration access token provides access to retrieve the client configuration later, but also to update or delete the client.
 The registration access token is included with the request in the same way as a bearer token or initial access token.
-Registration access tokens are only valid once, when it's used the response will include a new token. 
+
+By default, registration access token rotation is enabled. This means a registration access token is only valid once, when it's used the response will include a new token. Note that registration access token rotation can be disabled via link:{adminguide_link}#_client_policies[Client Policies].
```
Comment (andymunro) [inline securing_apps/topics/client-registration.adoc]:
```suggestion
By default, registration access token rotation is enabled. This means a registration access token is only valid once. When the token is used, the response will include a new token. Note that registration access token rotation can be disabled by using link:{adminguide_link}#_client_policies[Client Policies].
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good. Just a few suggestions.

**Review by mposolda — APPROVED** [review]
@reda-alaoui @andymunro Thanks for the documentation and review!

**Pull request merged**